app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -9,7 +9,7 @@ import Form from "react-bootstrap/Form";
 import InputGroup from "react-bootstrap/InputGroup";
 import { useCommonForm } from "../../../hooks";
 import { useConfig } from "../../../hooks/useConfig";
-import { EXTERNAL_URLS } from "../../../lib/constants";
+import { EXTERNAL_URLS, NUMBER_FIELD } from "../../../lib/constants";
```

Comment:
Thought maybe putting this in constants would be overkill, but a) they're used a lot and b) can't think of a better place so 👍

Reply:
Yeah, I originally had them living in `useCommonFormMethods` and didn't think `constants` was the _most_ ideal place for them, but after pushing and looking at my PR, I thought it was a better spot than where I had them originally due to how much they're used 🤔